shiny/express/ui/_cm_components.py

```diff
@@ -1153,6 +1165,7 @@ def value_box(
     theme: Optional[str | ui.ValueBoxTheme] = None,
     height: Optional[CssUnit] = None,
     max_height: Optional[CssUnit] = None,
+    min_height: Optional[CssUnit] = None,
```

Comment:
I think you need to pass this value to the body of the function?

Reply:
Good catch, thanks!